Resources/Prototypes/Nyanotrasen/Entities/Objects/Weapons/Guns/Pistols/pistols.yml

```diff
@@ -0,0 +1,41 @@
+- type: entity
+  name: mk 32 "Universal"
```

Comment:
Is the `Universal` necessary here? The mk 58 is just `mk 58`.

Reply:
As mentioned before, the mk 32 was intended to reference the Mk23/USP, thus the name "Universal".